packages/common/src/api/search.ts

```diff
@@ -80,11 +89,24 @@ const searchApi = createApi({
             bpmMin,
             bpmMax,
             key: formatMusicalKey(filters.key)
-          })
+          }
+
+          // Fire analytics only for the first page of results
+          if (offset === 0) {
+            dispatch(
+              make(Name.SEARCH_TAG_SEARCH, {
+                term: query,
+                source,
+                ...searchParams
+              })
```

Comment:
would we want to track the search category as well?

Reply:
Yup it's in there as `kind`!

Reply:
ah i see, cool 👍